appveyor.yml

```diff
@@ -0,0 +1,60 @@
+environment:
+  matrix:
+  - ARCH: "i686"
+    CCPATH: 'C:\mingw-w64\i686-6.3.0-posix-dwarf-rt_v5-rev1\mingw32\bin'
```

Comment:
using cygwin's gcc is only for compiling cygwin applications, which julia isn't. the cross compiler is x86_64-w64-mingw32-gcc